sdcm/cluster.py

```diff
@@ -2945,24 +2945,6 @@ def set_seed_flag(self, is_seed):
         if hasattr(self.log, 'extra'):
             self.log.extra['prefix'] = str(self)
 
-    def disable_daily_triggered_services(self):
```

Comment:
We need to consider some cases where we don't have cloud-init working

docker backend as an example, we might count on the docker build process to disable all of those.

I did run into the cases in artifact tests when we don't have cloud-init installed or it's disabled on some distros. and for them we repeat the startup script again, so it's covered as well

Reply:
but are docker images run these processes? I doubt as I think there's no even systemd there runnig.
I think this problem happens only in the cloud instances.

Reply:
I'm don't know if they do or don't, just suggesting we should validate it's not.

as for other distros, it might affect only artifact test, and we'll rarely see issues.
but it might also affect manager tests, that uses other distro for the monitoring for example.

it would be a bit of shame if we shove this part away, and it would start popping up across the board again.
that's the concern, and I wondering if we need to keep some fallback for those cases.

Reply:
I can apply similar trick like for syslog-ng - in case there's no marker file in dir it could run the old path. WDYT?

Reply:
actually, this is already applied if we use syslog-ng as stuff from `ConfigurationScriptBuilder` will be executed then anyway and skipped if already done.

Reply:
well this trick of redoing the cloud-init, kind of masquerade which platform/distro we are using supports cloud-init and which are not.
so it's hard to tell which one we need to test when changing things around cloud-init...

so I don't know even which one exactly to point that we should test...

Reply:
cloud-init is always available for AWS, GCE and Azure I suppose. So only cases we don't have it is docker and bare metal cases.

Reply:
it really depend on the image we are using, some images might not have cloud-init in them, or it's disabled by propose.
and we have quite vast of possibilities of images/distros we are using

Reply:
one more related think, we are not using syslog-ng for artifact tests, but ssh, partly cause of those issues (at least that what I think)

Reply:
ok, I see the point. I fixed the code to retry this disabling in case if was not disabled. Please review again.